### ISSUE TITLE
Not compatible with dbt 1.3.3

### ISSUE THREAD

**Issue by hermandr** [anelendata/dbt_docstring#3]
Error encountered
```
Traceback (most recent call last):
  File "/home/herman/.local/bin/dbtdocstr", line 8, in <module>
    sys.exit(main())
  File "/home/herman/.local/pipx/venvs/dbt-docstring/lib/python3.8/site-packages/dbt_docstring/__init__.py", line 132, in main
    _run(args.dbt_dir, args.backup)
  File "/home/herman/.local/pipx/venvs/dbt-docstring/lib/python3.8/site-packages/dbt_docstring/__init__.py", line 96, in _run
    models_dirs = _get_models_dirs(dbt_dir)
  File "/home/herman/.local/pipx/venvs/dbt-docstring/lib/python3.8/site-packages/dbt_docstring/__init__.py", line 20, in _get_models_dirs
    return config["source-paths"]
KeyError: 'source-paths'
```

dbt_project.yml extracted from https://docs.getdbt.com/reference/dbt_project.yml
```
[name](/reference/project-configs/name): string

[config-version](/reference/project-configs/config-version): 2
[version](/reference/project-configs/version): version

[profile](/reference/project-configs/profile): profilename

[model-paths](/reference/project-configs/model-paths): [directorypath]
[seed-paths](/reference/project-configs/seed-paths): [directorypath]
[test-paths](/reference/project-configs/test-paths): [directorypath]
[analysis-paths](/reference/project-configs/analysis-paths): [directorypath]
[macro-paths](/reference/project-configs/macro-paths): [directorypath]
[snapshot-paths](/reference/project-configs/snapshot-paths): [directorypath]
[docs-paths](/reference/project-configs/docs-paths): [directorypath]
[asset-paths](/reference/project-configs/asset-paths): [directorypath]

[target-path](/reference/project-configs/target-path): directorypath
[log-path](/reference/project-configs/log-path): directorypath
[packages-install-path](/reference/project-configs/packages-install-path): directorypath

[clean-targets](/reference/project-configs/clean-targets): [directorypath]

[query-comment](/reference/project-configs/query-comment): string

[require-dbt-version](/reference/project-configs/require-dbt-version): version-range | [version-range]

[quoting](/reference/project-configs/quoting):
  database: true | false
  schema: true | false
  identifier: true | false

models:
  [<model-configs>](/reference/model-configs)

seeds:
  [<seed-configs>](/reference/seed-configs)

snapshots:
  [<snapshot-configs>](/reference/snapshot-configs)

sources:
  [<source-configs>](source-configs)
  
tests:
  [<test-configs>](/reference/test-configs)

vars:
  [<variables>](/docs/build/project-variables)

[on-run-start](/reference/project-configs/on-run-start-on-run-end): sql-statement | [sql-statement]
[on-run-end](/reference/project-configs/on-run-start-on-run-end): sql-statement | [sql-statement]

[dispatch](/reference/project-configs/dispatch-config):
  - macro_namespace: packagename
    search_order: [packagename]

```

**Comment by aibunny** [comment]
I got the same error ```Traceback (most recent call last):
  File "/home/aibunny/venv/bin/dbtdocstr", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/aibunny/venv/lib/python3.12/site-packages/dbt_docstring/__init__.py", line 132, in main
    _run(args.dbt_dir, args.backup)
  File "/home/aibunny/venv/lib/python3.12/site-packages/dbt_docstring/__init__.py", line 96, in _run
    models_dirs = _get_models_dirs(dbt_dir)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aibunny/venv/lib/python3.12/site-packages/dbt_docstring/__init__.py", line 20, in _get_models_dirs
    return config["source-paths"]
           ~~~~~~^^^^^^^^^^^^^^^^
KeyError: 'source-paths'
```

**Comment by aibunny** [comment]
I get the same error 
```Traceback (most recent call last):
  File "/home/aibunny/venv/bin/dbtdocstr", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/aibunny/venv/lib/python3.12/site-packages/dbt_docstring/__init__.py", line 132, in main
    _run(args.dbt_dir, args.backup)
  File "/home/aibunny/venv/lib/python3.12/site-packages/dbt_docstring/__init__.py", line 96, in _run
    models_dirs = _get_models_dirs(dbt_dir)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aibunny/venv/lib/python3.12/site-packages/dbt_docstring/__init__.py", line 20, in _get_models_dirs
    return config["source-paths"]
           ~~~~~~^^^^^^^^^^^^^^^^
KeyError: 'source-paths'
```

**Comment by daigotanaka** [comment]
Sorry for the big delay. Just merged #4 
Big thx to @jbogaardt

**Comment by daigotanaka** [comment]
and 0.1.3 is available for install via PyPi
